src/pages/index.js

```diff
@@ -93,12 +120,12 @@ function handlePlaceCreateForm(data) {
     link: data.link,
   };
   placeCreateFormValidator.disableSubmit();
-
-  const card = createCard(cardData);
-  cardSection.addItem(card);
-  placeCreateForm.reset();
-  placeCreatePopup.close();
-  return false;
+  return api.createCard(cardData).then((card) => {
+    const cardElement = createCard(card);
+    cardSection.addItem(cardElement);
+    placeCreateForm.reset();
+    placeCreatePopup.close();
```

Comment:
You close and reset the modal inside `PopupWithForm`, so you don't need to duplicate it in `index.js`


**Please, correct it everywhere in the code**